include/net/nrf_cloud_alerts.h

```diff
@@ -78,7 +78,15 @@ struct nrf_cloud_alert_info {
  *
  * @return 0 if alert is successfully sent, otherwise, a negative error number.
  */
+#if defined(CONFIG_NRF_CLOUD_ALERTS) || defined(__DOXYGEN__)
```

Comment:
Ah, I was unaware of this technique.  Good to know.

Reply:
Ooh, this is neat, and perfectly solves some issues we've been having!